.gitignore

```diff
@@ -356,4 +356,7 @@ launchSettings.json
 local.settings.json
 src/PlayFabBuddy.Cli/MasterAccountOutput.json
 
-.vscode/
\ No newline at end of file
+.vscode/
+src/PlayFabBuddy.Cli/local.settings.json
+PlayFabBuddy.UI/local.settings.json
```

Comment:
Is this change necessary with line [356 ](https://github.com/Annonator/PlayFabBuddy/blob/bb99539eb343c4bcf6dd5a524e06d6bb474a0466/.gitignore#L356) in the gitignore?